lib/compare_task.rb

```diff
@@ -94,9 +97,13 @@ def render_comparison(description1, description2, scopes, options = {})
         identical = false if description1[scope] || description2[scope]
       end
     end
+    plural = "scopes are"
+    singular = "scope is"
+    count = identical_scopes.count
 
     output = "Compared descriptions are identical.\n" + output if identical && common_scopes
-
+    output += "Following #{Machinery::pluralize(count, singular, plural)} " \
+      "identical in both descriptions: " + identical_scopes.join(",") if !identical_scopes.empty?
```

Comment:
Even though this is for the interpreter only one line these are two for us so I would prefer `if` before the the output statement.